resources/assets/js/components/H2Components/Select.tsx

```diff
@@ -74,14 +77,16 @@ const Select: React.FunctionComponent<SelectContext> & SelectComposition = (
     children,
     ...rest
   } = props;
+  const registerProps = register ? register(name) : {};
+  console.log(defaultValue);
```

Comment:
@yonikid15 any reason to keep this console statement?

Reply:
Nope, one sec